test/verify/check-metrics

```diff
@@ -1327,17 +1327,12 @@ class TestMetricsPackages(packagelib.PackageCase):
         b.click(".pf-v5-c-empty-state button.pf-m-primary")
         b.click("#dialog button:contains('Install')")
         b.wait_not_present("#dialog")
-        b.click("button:contains('Log out')")
-        b.leave_page()
-        b.click("button:contains('Reconnect')")
-        b.set_val("#login-user-input", "admin")
-        b.set_val("#login-password-input", "foobar")
-        b.click('#login-button')
-        b.enter_page("/metrics")
         b.wait_in_text(".pf-v5-c-empty-state", "Metrics history could not be loaded")
         b.logout()
 
         # install c-pcp from the Metrics Settings dialog
+        # HACK: Removing the python PCP while it is running, will keep it
+        # working, as it is already loaded in memory.
```

Comment:
Not sure what is running here -- at this point the cockpit session is already logged out. Do you mean the daemons continue to run after uninstalling the packages? But there's no actual hack here to kill them. Or should this comment apply to the `b.logout()` above? (this PR doesn't touch that)

Reply:
Yeah hmm this is not an issue in the tests, but it is untested or "unfixed".

Reply:
Yes, I mean the pcp bridge keeps running once you removed it.